package.json

```diff
@@ -124,7 +124,8 @@
     "vue-svg-loader": "^0.16.0",
     "vue-template-compiler": "^2.7.7",
     "webpack-dev-middleware": "^5.3.3",
-    "webpack-node-externals": "^3.0.0"
+    "webpack-node-externals": "^3.0.0",
+    "volar-service-vetur": "latest"
```

Comment:
Why does this need to be installed? I'm not familiar with it and don't see it referenced anywhere in this changeset (it looks like it might be specifically for VSC, but I wouldn't think you'd need to install it in the project for it to work in VSC?)

Reply:
I had been using Vetur extension. While working on the codebase for this project and I received the prompt that Vetur extension should be removed and in place of it add Volar package.
I agree with your comment. I removed package and added Volar extension in my VS code and it worked fine.